tests/unit_tests/restruct_poc/test_math.cpp

```diff
@@ -0,0 +1,369 @@
+#include <gtest/gtest.h>
+
+#include "src/restruct_poc/math/quaternion_operations.hpp"
+#include "src/restruct_poc/math/vector_operations.hpp"
+#include "tests/unit_tests/gen_alpha_poc/test_utilities.h"
+
+namespace openturbine::restruct_poc::tests {
+
+TEST(QuaternionTest, ConvertQuaternionToRotationMatrix_1) {
```

Comment:
Is there a better name to use here?  Possibly ConvertQuaternionToRotationMatrix_XAxisRotation?